lib/runtime/life/instance.go

```diff
@@ -172,7 +195,7 @@ func (in *Instance) Exec(function string, data []byte) ([]byte, error) {
 
 	fnc, ok := in.vm.GetFunctionExport(function)
 	if !ok {
-		panic("entry function not found")
+		panic("entry function " + function + " not found")
```

Comment:
maybe change to return error instead of panic

Reply:
Yes, updated, here error seems better than panic.